src/main/java/io/github/mountainrange/mule/controllers/PlayController.java

```diff
@@ -27,7 +26,7 @@
 
 	private SceneLoader sceneLoader;
 	private MULE mule;
-	private VisualGrid<VisualTile> g;
+	private VisualGrid g;
 	private WorldMap map;
```

Comment:
Probably not `PlainTile`, since `PlayController` is in a JavaFX zone; we would need to pass the constructor to `WorldMap` a `PlainGrid` in that case. Try changing it to `WorldMap<Tile>` or `WorldMap<VisualTile>` and look at what the compiler says though.